### DISCUSSION TITLE
I got notification on new update of sandboxie plus, but does not see how to update with saving settings

### DISCUSSION THREAD

**Discussion by serpen7** [sandboxie-plus/Sandboxie#3629] [Q&A Feedback] [answered]
I got notification on new update but does not see how to update with saving settings. Can you help? What is the proper way to update sandboxie plus without loosing your settings and sandboxes?

**Comment by offhub** [comment, marked as answer]
You can download the [latest installer](https://github.com/sandboxie-plus/Sandboxie/releases/latest) and install it over the current one. Installing over it will keep your current settings. If you still want to save your settings, you can back up the following files.

To save your sandbox settings, just backup your `C:\Windows\Sandboxie.ini` file.

For Sandboxie-Plus interface settings, you need to save the `%LOCALAPPDATA%\Sandboxie-Plus\Sandboxie-Plus.ini` file.

> [!Note]
> For a portable installation, these files may be located in the folder where the application is running.

**Reply by serpen7** [reply]
Thank you I am using portable version and was in need to backup my settings before update cause I need to get settings into new portable version. When I opened it it was all on default so I thought it just deleted them.
